roles/openshift_aws/tasks/dns_route53_record.yml

```diff
@@ -0,0 +1,22 @@
+---
+- debug: msg="{{ openshift_aws_dns_records[l_openshift_aws_dns_element] }}"
+
+- name: querying elb
+  ec2_elb_facts:
+    names: "{{ openshift_aws_dns_records[l_openshift_aws_dns_element]['value'] }}"
+    region: "{{ openshift_aws_region }}"
+  register: elb_facts
+  when:
+    - "'elb' in openshift_aws_dns_records[l_openshift_aws_dns_element]['kind']"
```

Comment:
should be ==, not 'in'.  'in' implies a list, dictionary, or some long string that might contain our value.  We're definitely looking for it to be elb here, same for CNAME below.